usefulUtilities/userLocation/userLocation.js

```diff
@@ -0,0 +1,208 @@
+//
+//  userLocation.js
+//
+//  Created by Zach Fox on 2019-04-02
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  See accompanying README.md for usage instructions.
+// 
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+
+(function () {
+    var request = Script.require("./modules/request.js").request;
+
+    var _this;
+    var REFRESH_LOCATION_INTERVAL_MS = 30000;
+    var USER_API_BASE_URL = Account.metaverseServerURL + "/api/v1/users?filter=connections&search=";
+    var TEXT_LOCAL_ENTITY_PROPS = {
+        "type": "Text",
+        // eslint-disable-next-line no-magic-numbers
+        "localPosition": [0, 0, 0.05],
+        // eslint-disable-next-line no-magic-numbers
+        "localRotation": Quat.fromVec3Degrees([0, 0, 0]),
+        "lineHeight": 0.08,
+        // eslint-disable-next-line no-magic-numbers
+        "dimensions": [1.5, 0.2, 0.05],
```

Comment:
Ok I feel better about using arrays for vecs after seeing this, but taking away that it just depends on the context and if Milad can read it.

Reply:
👍